deploy/helm/flightctl/templates/flightctl-prometheus-deployment.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: apps/v1
```

Comment:
We should not install this on OCP, i.e.

{{ if .Values.prometheus.enabled }}
...
...